<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgHideWifiEapPeap</class>
 <widget class="QDialog" name="DlgHideWifiEapPeap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connect to Hidden WLAN Network</string>
  </property>
  <widget class="QLineEdit" name="leNetName">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>141</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxCA">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>341</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="lbConn">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>76</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="leDomain">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>301</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxConn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>71</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="leCaPwd">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>381</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConnect">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>650</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxSecurity">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>181</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="leAnonyId">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>261</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxCA">
   <property name="geometry">
    <rect>
     <x>183</x>
     <y>421</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbNetName">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>146</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>650</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbDomain">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>306</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbCaPwd">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>386</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxPwdSec">
   <property name="geometry">
    <rect>
     <x>357</x>
     <y>593</y>
     <width>18</width>
     <height>9</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxPwd">
   <property name="geometry">
    <rect>
     <x>357</x>
     <y>393</y>
     <width>18</width>
     <height>9</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbSecurity">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>186</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxPEAPver">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>461</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="lbAuth">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>226</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbPEAPver">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>466</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxAuth">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>221</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="lbCA">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>346</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbAnonyId">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>266</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbPassword">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>586</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="leUserName">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>541</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxInnerAuth">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>501</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePassword">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>581</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbInnerAuth">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>506</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbUserName">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>546</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbBoder">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>432</width>
     <height>700</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbLeftupTitle">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>28</y>
     <width>140</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="lineUp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>121</y>
     <width>412</width>
     <height>1</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="lineDown">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>631</y>
     <width>412</width>
     <height>1</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <zorder>lbBoder</zorder>
  <zorder>leNetName</zorder>
  <zorder>cbxCA</zorder>
  <zorder>lbConn</zorder>
  <zorder>leDomain</zorder>
  <zorder>cbxConn</zorder>
  <zorder>leCaPwd</zorder>
  <zorder>btnConnect</zorder>
  <zorder>cbxSecurity</zorder>
  <zorder>leAnonyId</zorder>
  <zorder>checkBoxCA</zorder>
  <zorder>lbNetName</zorder>
  <zorder>btnCancel</zorder>
  <zorder>lbDomain</zorder>
  <zorder>lbCaPwd</zorder>
  <zorder>checkBoxPwd</zorder>
  <zorder>lbSecurity</zorder>
  <zorder>cbxPEAPver</zorder>
  <zorder>lbAuth</zorder>
  <zorder>lbPEAPver</zorder>
  <zorder>cbxAuth</zorder>
  <zorder>lbCA</zorder>
  <zorder>lbAnonyId</zorder>
  <zorder>lbPassword</zorder>
  <zorder>leUserName</zorder>
  <zorder>cbxInnerAuth</zorder>
  <zorder>lePassword</zorder>
  <zorder>lbInnerAuth</zorder>
  <zorder>lbUserName</zorder>
  <zorder>checkBoxPwdSec</zorder>
  <zorder>lbLeftupTitle</zorder>
  <zorder>lineUp</zorder>
  <zorder>lineDown</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
